src/Pages/AddItem.js

```diff
@@ -72,3 +79,19 @@ export default function AddItem({ token }) {
     </div>
   );
 }
+const validateNewListItem = (listItem, list) => {
+  const punctuation = /[!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~]/g;
+
+  const filteredList = list.docs.filter((filteredItem) => {
+    const existingItem = filteredItem
+      .data()
+      .item.toLowerCase()
+      .replace(punctuation, '');
+
+    const newItem = listItem.toLowerCase().replace(punctuation, '');
+
+    return existingItem === newItem;
+  });
+
+  return filteredList.length > 0 ? false + console.log('Error!') : true;
```

Comment:
Hmm.. does this work? I would expect there to be some kind of error.. I think it would be best to get rid of our inclusion of the console log here. Best part is,  if we do that, we don't even need the ternary! We can just return the expression `filteredList.length > 0`

Reply:
I updated line 96 with the double bang operator!!